tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -4539,7 +4569,7 @@ uint32_t SPIRVEmitter::processIntrinsicClip(const CallExpr *callExpr) {
         theBuilder.createCompositeConstruct(boolRowType, cmpResults);
     condition = theBuilder.createUnaryOp(spv::Op::OpAny, boolType, results);
   } else {
-    emitError("Invalid type passed to clip function.");
+    emitError("invalid type passed to clip function", callExpr->getExprLoc());
```

Comment:
invalid argument type passed to clip intrinsic function?

Reply:
Done